.github/workflows/gradle.yml

```diff
@@ -0,0 +1,23 @@
+
+on:
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  gradle:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-java@v4
+      with:
+        distribution: temurin
+        java-version: 17
```

Comment:
In other areas of our project we have specified the Java version to be 21, is it supposed to be 17 here?

Reply:
My bad, missed that